db/db.login/db.login.html

```diff
@@ -15,8 +15,8 @@ <h2>NOTE</h2>
 file in the user account, specifically
 
 <ul>
-<li> in the 'home' directory, i.e. <tt>$HOME/.grass7/dblogin</tt> (Unix-like systems)</li>
-<li> <tt>%APPDATA%/GRASS7/dblogin</tt> (MS-Windows)</li>
+<li> in the 'home' directory, i.e. <tt>$HOME/.grass8/dblogin</tt> (Unix-like systems)</li>
+<li> <tt>%APPDATA%\Roaming\GRASS8\dblogin</tt> (MS-Windows)</li>
```

Comment:
Doesn't `/` just work on Windows [these days](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.altdirectoryseparatorchar?view=net-5.0), @hellik?

Reply:
/ works on Windows (https://answers.microsoft.com/en-us/windows/forum/windows_7-files/when-did-windows-start-accepting-forward-slash-as/aa033741-0fc6-43e3-ab6e-165020794126), but I don't think it's a good idea to use / in general (https://bytes.com/topic/python/answers/23123-when-did-windows-start-accepting-forward-slash-path-separator), not in this specific case.
```dos
cd /users # works fine
mkdir \dev
cd /dev # doesn't work because /d is an option
```

Reply:
> cd /data  # doesn't work because /d is an option

AFAIK it's 

`cd data`

`cd \data` changes here to an existing directory in the toplevel d: , altough I've started e.g. in D:\temp\tm18> where the data subfolder exists.

```
D:\temp\tm18>cd /data
Das System kann den angegebenen Pfad nicht finden.
```

Reply:
> 
> 
> Doesn't `/` just work on Windows [these days](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.altdirectoryseparatorchar?view=net-5.0), @hellik?

the interesting thing is - and almost nobody knows it ;-)  - it worked since the beginning of MS windows

```
D:\>cd D:\temp\tm18\data
D:\temp\tm18\data>
```

```
D:\>cd D:/temp/tm18/data
D:\temp\tm18\data>
```

though the MS windows standard, everywhere promoted, is:

`D:\temp\tm18\data`

and you can't be sure that it works in all circumstances of myriads of different varieties of batch files, python scripts, R scripts in windows .....

Reply:
I wonder if there are any circumstances where it is still an issue except for that `/d` issue. Python scripts should never have explicit slash or backslash in them. The best practices for R I have seen argued for forward slashes on all platforms (saying to make them work on macOS and Linux, so they actually seemed written from Windows point of view).